test/fixtures/consola-with-tag/nuxt.config.js

```diff
@@ -0,0 +1,11 @@
+import consola from 'consola'
+
+module.exports = {
+  hooks: {
+    build: {
+      done () {
+        consola.withTag('build').error('bar')
```

Comment:
We can move both to basic fixture to avoid building/setupTest overhead

Reply:
Updated